src/components/Categories/Categories.js

```diff
@@ -1,16 +1,64 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import styled from 'styled-components/macro';
+import Products from '../../data/data.json';
 
 function Categories(props) {
+    const [products, setProducts] = useState([]);
+
+    useEffect(() => {
+        let items = Products.filter(
+            (Product) => Product.category.title === props.match.params.slug
+        );
+        setProducts(items);
+    }, [props.match.params.slug]);
```

Comment:
You are using the `props.match.params.slug` twice. Maybe it would be better to save it as a variable?
```suggestion
    const slug = props.match.params.slug

    useEffect(() => {
        const items = Products.filter(
            (Product) => Product.category.title === slug
        );
        setProducts(items);
    }, [slug]);
```

Reply:
its better. thanx